plug-ins/groups/class_antipaladin.cpp

```diff
@@ -258,7 +259,7 @@ void ShadowBlade::fight( Character *ch )
     if (obj->wear_loc != wear_wield && obj->wear_loc != wear_second_wield) 
         return;
     
-    level = ch->getModifyLevel( );
+    level = skill_level(*gsn_shadowblade, ch);
```

Comment:
@fiorine @Kerrad this line looks more like an anti-bonus to me. If previously for lvl 100 A-P it was enough to fight victims of level `90` and above, now with `+3` skill level bonus the victim has to have level `93` and above, otherwise the blade won't improve.